python/paddle/incubate/nn/functional/__init__.py

```diff
@@ -109,12 +110,12 @@
     "masked_multihead_attention",
     "blha_get_max_len",
     "block_multihead_attention",
-    "swiglu",
     "moe_combine",
     "expand_modality_expert_id",
     "cal_aux_loss",
     "build_src_rank_and_local_expert_id",
     "int_bincount",
+    "swiglu",
```

Comment:
because we no longer recommend using `paddle.incubate.nn.functional.swiglu` in the future, it can be removed from \_\_all\_\_ list here